test/system/050-stop.bats

```diff
@@ -57,8 +57,8 @@ load helpers
 
         # The 'stop' command should return almost instantaneously
         delta_t=$(( $t1 - $t0 ))
-        [ $delta_t -le 2 ] ||\
-            die "podman stop: took too long ($delta_t seconds; expected <= 2)"
+        [ $delta_t -le 3 ] ||\
+            die "podman stop: took too long ($delta_t seconds; expected <= 3)"
```

Comment:
The changes here seem unrelated and should be in a separate commits explaining why they're needed.

Reply:
Honestly I have no idea, do you saw this in other PRs failing as well? I constantly had a higher delay in that test... :man_shrugging: